Source/WebCore/html/PluginDocument.cpp

```diff
@@ -62,19 +66,44 @@ class PluginDocumentParser final : public RawDataDocumentParser {
 
     void appendBytes(DocumentWriter&, const uint8_t*, size_t) final;
     void createDocumentStructure();
+    Ref<HTMLStyleElement> createStyleElement(Document&);
 
     WeakPtr<HTMLEmbedElement, WeakPtrImplWithEventTargetData> m_embedElement;
 };
 
+Ref<HTMLStyleElement> PluginDocumentParser::createStyleElement(Document& document)
+{
+    auto styleElement = HTMLStyleElement::create(document);
+
+    constexpr auto styleSheetContents = R"CONTENTS(
```

Comment:
what's `CONTENTS` mean here?

Reply:
It's the arbitrary delimiter for the string literal. We also have `DL`/`PSONRESOURCE`/etc.